extension/src/popup/components/sendPayment/SendAmount/index.tsx

```diff
@@ -119,10 +120,38 @@ export const SendAmount = ({ previous }: { previous: ROUTES }) => {
     destinationAsset,
   } = transactionData;
 
+  const { recommendedFee } = useNetworkFees();
+
+  const calculateAvailBalance = useCallback(
```

Comment:
this fixes 3 issues:

1) we were deducting base reserve from non-XLM assets. Now we only do that for XLM
2) we take into account the recommendedFee when showing what your Max available to send is (otherwise, you could click Send Max, try to submit to Horizon, and it would fail bc the fee would put you over your limit)
3) we centralize this "Max Available" logic so in the header it shows the same amount that will be populated into the field when you click "Send Max"